mitiq/zne/inference.py

```diff
@@ -554,7 +551,7 @@ def run(
 
         # If there are different keyword args, run each circuit individually.
         # https://stackoverflow.com/questions/1151658/python-hashable-dicts.
-        class HashableDict(Dict[Any, Any]):
+        class HashableDict(dict[Any, Any]):
```

Comment:
```suggestion
        class HashableDict(dict):
```
Do you know, is this the same as having `dict[Any, Any]`?

Reply:
It is the same. Just that it's better to have `dict[Any, Any]`. Pyright will report `x: dict` in strict mode because it contains `Unknown`, that's the difference. It has the same meaning as `Any` for other purposes.

Reply:
Got it. As I'm sure you're aware, we are currently typechecking with `mypy` which may give a different response?